artifactory_cleanup/rules/keep.py

```diff
@@ -177,7 +182,13 @@ def filter(self, artifacts):
             if good_artifact_count < 0:
                 good_artifact_count = 0
 
-            good_artifacts = artifactory_with_version[good_artifact_count:]
+            good_sets = artifactory_with_version[good_artifact_count:]
+
+            good_artifacts = []
```

Comment:
Let's write it as list comprehension
```python
# artifactory_with_version contains list of (Version, Artifact)  pairs
# Get the artifacts from that
good_sets = artifactory_with_version[good_artifact_count:]
good_artifacts = [good[1] for good in good_sets]
```

Could you also add a comment about why do we need that step - just as explained in the issue #80 but shorter.

Reply:
Good suggestion that is a bit cleaner. Sorry for not doing that to start my python is a bit rusty.

Reply:
Should be addressed now. Added a bit more to your comment. Please let me know if you would like more detail in the code comment.

Reply:
>Sorry for not doing that to start my python is a bit rusty.

No need to apologize! It's why we're here to guide you.